src/mono/wasm/runtime/debug.ts

```diff
@@ -158,10 +158,19 @@ export function mono_wasm_wait_for_debugger(): Promise<void> {
 }
 
 export function mono_wasm_debugger_attached(): void {
-    runtimeHelpers.wait_for_debugger = 1;
+    if (runtimeHelpers.wait_for_debugger == -1)
+        runtimeHelpers.wait_for_debugger = 1;
     cwraps.mono_wasm_set_is_debugger_attached(true);
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function mono_wasm_set_entrypoint_breakpoint(assembly_name: CharPtr, token: number): void {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const assembly_name_str = assembly_name !== CharPtrNull ? Module.UTF8ToString(assembly_name).concat(".dll") : "";
```

Comment:
If we don't have an assembly name here, then can skip triggering the debugger?

Reply:
done